common/jmeter/saml/SAML2_SSO_Redirect_Binding.jmx

```diff
@@ -368,6 +388,11 @@ ${password}
 </stringProp>
         </HTTPSamplerProxy>
         <hashTree>
+          <GaussianRandomTimer guiclass="GaussianRandomTimerGui" testclass="GaussianRandomTimer" testname="Gaussian Random Timer" enabled="true">
```

Comment:
why we need this change ?

Reply:
As we discussed in the meetings, add a random delay to imitate the user login delay to input credentials.
**mail thread** - Presenting our performance results in an optimal way to do capacity planning for our customers